### PR TITLE
coffee

### PR DESCRIPTION
**Pull request by valeIT**

### CODE REVIEW

File: _posts/2019-11-08-coffee.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+type: post
+published: true
+status: publish
+categories: [General]
+image:
+image2:
+author: Valentino Urbano
+---
+
+There are a lot of different kind of coffees, some are good, others just so so, and then there are the truly awful ones that make you wonder if whoever produced it even knew what coffee is.
```
Comment (MsSlacker) [inline _posts/2019-11-08-coffee.md]: Don't think it's countable? So no s?

File: _posts/2019-11-08-coffee.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+Being italian I have a pretty high quality standard for coffee so it has always been hard to find one that I really liked abroad. Fortunately in most Europe you can find Italian branded coffee, it usually comes with a premium, but it is worth the difference in price.
```
Comment (MsSlacker) [inline _posts/2019-11-08-coffee.md]: Big I!

File: _posts/2019-11-08-coffee.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+If that's not an option the last line of defence is adding milk. It makes even terrible coffee way more bearable. It doesn't make it good, but it just make it so I can drink it without being disgusted by it too much.
+
+If that's not enough to save it that's it, time to only drink tea for a while.
+
+If I can drink it whole without internally complaining it means it's actually quite good. Honestly I haven't found many of that quality abroad, only a handful of times.
```
Comment (MsSlacker) [inline _posts/2019-11-08-coffee.md:24]: No mention of terrible ones? Like the choco one you said about earlier?

**Review by MsSlacker — CHANGES_REQUESTED** [review]

File: _posts/2019-11-08-coffee.md (hunk 1 above)
Comment (valeIT) [reply _posts/2019-11-08-coffee.md]: Yeah, sounds bad

**Review by MsSlacker — APPROVED** [review]

**Pull request merged**